e2e/puppeteer-custom-environment.ts

```diff
@@ -59,8 +59,6 @@ class PuppeteerCustomEnvironment extends PuppeteerEnvironment {
           await this.takeScreenshot(screenshotFile);
           const htmlFile = `${this.htmlDir}/${testName}.html`;
           await this.savePageToFile(htmlFile);
-          const pdfFile = `${this.htmlDir}/${testName}.pdf`;
-          await this.savePdf(pdfFile);
```

Comment:
Saving pdf feature is not supported in headful mode. Remove it so it doesn't interfere with run on localhost.